tests/t501-operator-f.f

```diff
@@ -0,0 +1,171 @@
+      call ceedelemrestrictioncreateidentity(ceed,nelem,2,2*nelem,1,
+     $  erestrictxi,err)
+
+      do i=0,nelem-1
+        do j=0,p-1
+          indu(p*i+j+1)=i*(p-1)+j
+        enddo
+      enddo
+
+      call ceedelemrestrictioncreate(ceed,nelem,p,nu,1,
+     $  ceed_mem_host,ceed_use_pointer,indu,erestrictu,err)
+      call ceedelemrestrictioncreateidentity(ceed,nelem,q,q*nelem,1,
+     $  erestrictui,err)
+
+      call ceedbasiscreatetensorh1lagrange(ceed,1,1,2,q,ceed_gauss,
+     $  bx,err)
+      call ceedbasiscreatetensorh1lagrange(ceed,1,1,p,q,ceed_gauss,
+     $  bu,err)
+
+      call ceedqfunctioncreateinterior(ceed,1,setup,
+c     __FILE__ should not be more than the 72 characters, -ffree-line-length-none ?
+     $__FILE__ 
+     $     //':setup'//char(0),qf_setup,err)
+c     $  't30-operator-f.f:setup',qf_setup,err)
+      call ceedqfunctionaddinput(qf_setup,'_weight',1,
+     $  ceed_eval_weight,err)
+      call ceedqfunctionaddinput(qf_setup,'x',1,ceed_eval_grad,err)
+      call ceedqfunctionaddoutput(qf_setup,'rho',1,
+     $  ceed_eval_none,err)
+
+      call ceedqfunctioncreateinterior(ceed,1,mass,
+     $__FILE__ 
+     $     //':mass'//char(0),qf_mass,err)
+c     $  't30-operator-f.f:mass',qf_mass,err)
+      call ceedqfunctionaddinput(qf_mass,'rho',1,ceed_eval_none,err)
+      call ceedqfunctionaddinput(qf_mass,'u',1,ceed_eval_interp,err)
+      call ceedqfunctionaddoutput(qf_mass,'v',1,ceed_eval_interp,err)
+
+      call ceedoperatorcreate(ceed,qf_setup,ceed_null,ceed_null,
+     $  op_setup,err)
+      call ceedoperatorcreate(ceed,qf_mass,ceed_null,ceed_null,
+     $  op_mass,err)
+
+      call ceedvectorcreate(ceed,nx,x,err)
+      call ceedvectorsetarray(x,ceed_mem_host,ceed_use_pointer,arrx,err)
+      call ceedvectorcreate(ceed,nelem*q,qdata,err)
+
+      call ceedoperatorsetfield(op_setup,'_weight',
+     $  erestrictxi,bx,ceed_vector_none,err)
+      call ceedoperatorsetfield(op_setup,'x',erestrictx,bx,
+     $  ceed_vector_active,err)
+      call ceedoperatorsetfield(op_setup,'rho',
+     $  erestrictui,ceed_basis_colocated,
+     $  ceed_vector_active,err)
+      call ceedoperatorsetfield(op_mass,'rho',
+     $  erestrictui,ceed_basis_colocated,
+     $  qdata,err)
+      call ceedoperatorsetfield(op_mass,'u',erestrictu,bu,
+     $  ceed_vector_active,err)
+      call ceedoperatorsetfield(op_mass,'v',erestrictu,bu,
+     $  ceed_vector_active,err)
+
+      call ceedoperatorapply(op_setup,x,qdata,
+     $  ceed_request_immediate,err)
+
+      call ceedvectorcreate(ceed,nu,u,err)
+      call ceedvectorsetvalue(u,1.,err)
+      call ceedvectorcreate(ceed,nu,v,err)
+      call ceedoperatorapply(op_mass,u,v,ceed_request_immediate,err)
+
+      call ceedvectorgetarrayread(v,ceed_mem_host,hv,err)
+      total=0.
+      do i=1,nu
+        total=total+hv(i)
+      enddo
+c      if (abs(total-1.)>1.0D-10) then
+c        write(*,'(A,F12.8,A)')
+c     $  'Computed Area: ',total,' != True Area: 1.0'
+c      endif
```

Comment:
'getarrayread' in Fortran does not appear to be working correctly. I wonder if PR #134 will fix this.